src/CTA.WebForms2Blazor/FileConverters/ViewFileConverter.cs

```diff
@@ -3,31 +3,39 @@
 using System.IO;
 using System.Linq;
 using System.Text;
+using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using CTA.Rules.Config;
 using CTA.WebForms2Blazor.ControlConverters;
 using CTA.WebForms2Blazor.FileInformationModel;
 using CTA.WebForms2Blazor.Helpers;
 using CTA.WebForms2Blazor.Helpers.ControlHelpers;
+using CTA.WebForms2Blazor.Metrics;
 using CTA.WebForms2Blazor.Services;
 using HtmlAgilityPack;
 
 namespace CTA.WebForms2Blazor.FileConverters
 {
     public class ViewFileConverter : FileConverter
     {
+        private const string ChildActionType = "ViewFileConverter";
+        private const string UnSupportedControlConverter = "UnSupportedControlConverter";
+        private Regex AspControlsRegex = new Regex(@"<asp:(\w+)\s+.*?id=(\w +)");
```

Comment:
Still a few slight issues with this regex, for an unsupported control `node.Name` below would look something like this: _asp:unsupportedcontrol_ but it might also look like _yyy:othercontrol_. To match this, a simple expression like `\w+:\w+` would do fine. I'm not sure if the `?id=(\w+)` is meant to be a capture group but if so then the expression could be modified to look like this `\w+:(?<id>\w+)`